core-java/src/main/java/org/baeldung/equalshashcode/entities/ComplexClass.java

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
I will move org.baeldung.* code to com.baeldung.* in the new module, so that we have everything under the same base package